frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/experiment-design.component.html

```diff
@@ -75,31 +75,32 @@
             </mat-cell>
           </ng-container>
 
-          <!-- Required Column -->
-          <!-- <ng-container matColumnDef="requiredId">
+          <!-- ExcludeIfReached Column -->
+           <ng-container matColumnDef="excludeIfReached">
             <mat-header-cell class="ft-14-700" *matHeaderCellDef>
-              {{ 'home.new-experiment.design.required.text' | translate }}
+              {{ 'home.new-experiment.design.exclude-if-reached.text' | translate }}
             </mat-header-cell>
             <mat-cell
               *matCellDef="let element; let groupIndex = index"
               [formGroupName]="groupIndex"
-              style="margin-left: 25px;"
+              style="margin-left:130px;"
             >
               <mat-checkbox
-                      class="ft-8-100"
-                      color="primary"
-                      [disabled]="true"
-                      >
+              class="ft-8-100"
+              color="primary"
+              [disabled]= "experimentInfo && (experimentInfo.state == ExperimentState.ENROLLING || experimentInfo.state == ExperimentState.ENROLLMENT_COMPLETE)"
+              [checked]="this.partition.at(groupIndex).get('excludeIfReached').value"
+              (change)="setExcludeIfReachedFlag($event,groupIndex)"
```

Comment:
Is this checkbox not registered as part of the form? we shouldn't have to manually handle setting the form value, something seems off with a (change) handler in a reactive form pattern. is there something special we're doing with this case?

Reply:
meaning if you use `[formControlName] = "excludeIfReached"`, I don't think a `(change)` should be needed?

Reply:
Right, we can do it that way to make it more simpler. I will update the PR. :)